registry/server/common/services/json.ts

```diff
@@ -13,23 +13,23 @@ const isJSON = (str: string): boolean => {
     return (/^[\],:{}\s]*$/).test(str);
 };
 
-const parse = (value: any): any => {
+const parseJSON = (value: any): any => {
```

Comment:
Why do you need to rename all these vars which unnecessarily cause a lot of side changes?

Reply:
Removed